### PR TITLE
Add verbose flag

### LINKED ISSUE #72: Add verbose output
Add CLI option to turn on verbose output and also a method that write lines to output if verbose is enabled.

### PR DESCRIPTION
**Pull request by bhirsz**
Closes #72

### CODE REVIEW

File: robocop/config.py [hunk 1]
```diff
@@ -140,19 +144,20 @@ def __init__(self, root=None, from_cli=False):
...
-        'help_argfile':      'Path to file with arguments',
-        'help_ignore':       'Ignore file(s) and path(s) provided. Glob patterns are supported',
-        'help_info':         'Print this help message and exit',
-        'help_version':      'Display Robocop version',
+        'help_recursive':    'Use this flag to stop scanning directories recursively.',
+        'help_argfile':      'Path to file with arguments.',
+        'help_ignore':       'Ignore file(s) and path(s) provided. Glob patterns are supported.',
+        'help_info':         'Print this help message and exit.',
+        'help_version':      'Display Robocop version.',
+        'verbose':           'Display extra information.',
```
Comment (mnojek) [inline robocop/config.py:160]: Let's be consistent
```suggestion
        'help_verbose':           'Display extra information.',
```

File: robocop/config.py [hunk 2]
```diff
@@ -245,32 +250,40 @@ def _create_parser(self):
         optional.add_argument('-h', '--help', action='help', help=self.HELP_MSGS['help_info'])
         optional.add_argument('-v', '--version', action='version', version=__version__,
                               help=self.HELP_MSGS['help_version'])
+        optional.add_argument('--verbose', action='store_true', help=self.HELP_MSGS['verbose'])
```
Comment (mnojek) [inline robocop/config.py:253]:
```suggestion
        optional.add_argument('--verbose', action='store_true', help=self.HELP_MSGS['help_verbose'])
```

File: robocop/config.py [hunk 3]
```diff
@@ -245,32 +250,40 @@ def _create_parser(self):
...
         args = self.preparse(args) if from_cli else None
-        if not args:
-            args = self.load_default_config_file()
-        if not args:
+        if not args or args == ['--verbose']:
+            loaded_args = self.load_default_config_file()
+            if loaded_args is not None:
+                # thanks for this we can have config file together with some cli options like --verbose
+                args = [*args, *loaded_args] if args is not None else loaded_args
+        if not args or args == ['--verbose']:
```
Comment (mnojek) [inline robocop/config.py:266]: This is a duplicate of the condition above

Comment (bhirsz) [reply robocop/config.py:266]: It's not - args can be empty in the line 261 (no cli arguments, or only --verbose) but then there could be .robocop file with arguments. Depending if we were able to find .robocop file, args could be still empty or not. That's why we need to check for it again before looking for pyproject.toml

Comment (bhirsz) [reply robocop/config.py:266]: But when I think about it, I think load.pyproject_file should be called only if loaded_args is None - we will avoid the situation where there .robocop with only --verbose inside.

**Pull request merged**